pkg/apis/ark/v1/labels_annotations.go

```diff
@@ -40,4 +40,8 @@ const (
 	// StorageLocationLabel is the label key used to identify the storage
 	// location of a backup.
 	StorageLocationLabel = "ark.heptio.com/storage-location"
+
+	// ForNamespaceLabel is the label key used to identify which namespace
+	// a restic repository stores pod volume backups for.
+	ForNamespaceLabel = "ark.heptio.com/for-namespace"
```

Comment:
This feels a bit too generic - can we include something that indicates this is for restic? Also, can we remove "for"?